client/trino-client/src/main/java/io/trino/client/FixJsonDataUtils.java

```diff
@@ -217,61 +217,19 @@ private static ColumnTypeHandler createTypeHandler(ClientTypeSignature signature
             case ROW:
                 return new RowClientTypeHandler(signature);
             case BIGINT:
-                return (value) -> {
-                    if (value instanceof String) {
-                        return Long.parseLong((String) value);
-                    }
-                    return ((Number) value).longValue();
-                };
-
+                return BigintTypeHandler.INSTANCE;
```

Comment:
What is the difference? I would expect the lambda is inlined into static code anyway.

Reply:
I don't want JVM spending time trying to figure out whether lambda can be inlined or not

Reply:
You should. It is JVMs job. If it you do not know it is explicitly needed, prooved by benchmarks please revert.